test_track_rails_client.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.add_dependency "faraday", ">= 0.8", "< 1.0"
   s.add_dependency 'faraday_middleware'
   s.add_dependency 'mixpanel-ruby', '~> 1.4'
+  s.add_dependency 'multi_json', '~> 1.7'
```

Comment:
Is this where you landed? We need this to reliably support vendored Her gem if nothing else depends on it?

Reply:
yup...the same failure came up on CI before i added this, so it does seem like some transitive dependency disappeared, requiring us to now explicitly define it.